app/template/[templateId]/section/[section_slug]/page.tsx

```diff
@@ -2,25 +2,46 @@
 
 import React from 'react';
 import {
+  Breadcrumb,
+  Breadcrumbs,
   Button,
   FieldError,
   Form,
   Input,
   Label,
+  Link,
   Tab,
   TabList,
   TabPanel,
   Tabs,
   Text,
   TextField
 } from "react-aria-components";
+import PageHeader from "@/components/PageHeader";
 
 const SectionEditPage: React.FC = () => {
   // For a real application, you'd fetch the existing section data using the sec_hash
 
   return (
     <>
-      <h1>Edit Section</h1>
+
```

Comment:
I noticed that when I get to the Edit Section page, and I tab, I cannot tab to the other tabs, like "Options" and "Logic"